.rubocop.yml

```diff
@@ -98,3 +98,5 @@ Layout/EmptyLineBetweenDefs:
 Layout/FirstHashElementIndentation:
   Enabled: false
 
+Lint/Syntax:
```

Comment:
This doesn't make sense to me. https://docs.rubocop.org/rubocop/cops_lint.html#lintsyntax only guarantees it's valid Ruby syntax. Why would you disable this?